spec/rspec_use_of_tags_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'support/helper'
+
+RSpec.describe "rspec can take tags to identify tests", :first do
+  it "can see the tags and the correct tests will run" do
+    expect(5).to eq(5)
+  end
+  
+  it "will also see individual tags", :single do
+    expect(1).to eq(1)
+  end
+end
+
+RSpec.describe "will separate tests in rspec to it and describe blocks", :second do 
+  it "will be able to differentiate between different blocks" do
+    expect("rspec is cool").to eq("rspec is cool")
+  end
+end
```

Comment:
How about adding this file to the cucumber features. Then you can invoke it with each of the different tags and assert that you see what you expect. The advantage of this is that the test exercises the tagging functionality (unless it does do this and I just don't see it anywhere). Maybe just before  [this](https://github.com/JoshCheek/mrspec/blob/278f0489119ebb31466750a91bd4412374654026/features/mrspec.feature#L275) test, which would show how the RSpec behaviour works, or in addition to that test to show that the tags get consolidated (actually, that test is already long, so if you want to do that one, then probably break it out into a separate Scenario).